src/main/kotlin/lotto/domain/Statistics.kt

```diff
@@ -0,0 +1,28 @@
+package lotto.domain
+
+import kotlin.math.round
+
+object Statistics {
+
+    var count_3 = 0
+    var count_4 = 0
+    var count_5 = 0
+    var count_6 = 0
+
+    fun countRank(winningCount: Int) {
+        when (winningCount) {
+            3 -> count_3++
+            4 -> count_4++
+            5 -> count_5++
+            6 -> count_6++
+        }
+    }
+
+    fun winningAmount(): Int {
+        return (count_3 * 5000) + (count_4 * 50000) + (count_5 * 1500000) + (count_6 * 2000000000)
```

Comment:
`100_000_000`처럼 숫자에 밑줄을 사용하여 가독성을 높일 수 있어요.

```suggestion
        return (count_3 * 5_000) + (count_4 * 50_000) + (count_5 * 1_500_000) + (count_6 * 2_000_000_000)
```